site/documentation/content/user-guide/coap-adapter.md

```diff
@@ -0,0 +1,231 @@
+  * (optional) `location-query: hono-command`: The name of the command to execute. This header will only be present if the response contains a command to be executed by the device.
+  * (optional) `location-path: ('command'|'command_response/'<command-request-id>)`: The location path contains `'command'` for one-way-commands, or `'command_response' / <command-request-id>`  for commands expecting  a response. In case of a expected response, the location-path contains exactly the URI-path expected for the response.
+* Response Body:
+  * (optional) Arbitrary data serving as input to a command to be executed by the device, if status code is 200.
+  * (optional) Error details, if status code is >= 400.
+* Status Codes:
+  * 205 (CONTENT): The telemetry data has been accepted for processing. The response contains a command for the device to execute.
+  * 204 (CHANGED): The telemetry data has been accepted for processing. Note that if the message type is `NON` (*at most once* semantics),
+    this status code does **not** mean that the message has been delivered to any potential consumer. However, if the message type is `CON` (*at least once* semantics), then the adapter waits for the message to be delivered and accepted by a downstream consumer before responding with
+    this status code.
+  * 400 (Bad Request): The request cannot be processed. Possible reasons for this include:
+        * The content type header is missing.
+        * The request body is empty.
+        * The QoS header value is invalid.
+  * 401 (Unauthorized): The request cannot be processed because the request does not contain valid credentials.
+  * 403 (Forbidden): The request cannot be processed because the device's registration status cannot be asserted. Possible reasons for this include:
+        * The given tenant is not allowed to use this protocol adapter.
+  * 404 (Not Found): The request cannot be processed because the device is disabled or does not exist.
+  * 429 (Too Many Requests): The request cannot be processed because the tenant's message limit for the current period is exceeded.
+  * 503 (Service Unavailable): The request cannot be processed because there is no consumer of telemetry data for the given tenant connected to Hono.
+
+This is the preferred way for devices to publish telemetry data. It is available only if the protocol adapter is configured to require devices to authenticate (which is the default).
+
+If the `hono-ttd` URI-query option is set in order to receive a command and if the authenticated device is actually a gateway, the returned command will be the first command that the northbound application has sent to either the gateway itself or (if not using the legacy control endpoint) to *any* device that has last sent a telemetry or event message via this gateway.
```

Comment:
there is no legacy control endpoint anymore ...

Reply:
Please propose the right section (possibly update http-guide?).

Reply:
???

Reply:
I have no glue, what this part was good for. And I don't know, what should be written instead. Hono has quite a lot implemented in the base-classes, and I don't know, what is changed there  or not.

Reply:
just remove the
> (if not using the legacy control endpoint)

Reply:
OK